app/src/main/java/zafrani/com/pubgapp/models/Category.java

```diff
@@ -2,25 +2,49 @@
 
 
 import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
 
 import com.google.gson.annotations.SerializedName;
 
+import java.util.List;
+
 public class Category {
     @SerializedName("cat_name")
-    private String name;
+    private String cat_name;
```

Comment:
this should just be name. Its obviously a category name becauses its a name inside a category object.